NGitLab/PublicAPI.Unshipped.txt

```diff
@@ -2818,6 +2793,7 @@ NGitLab.Models.UserToken.CreatedAt.get -> System.DateTime
 NGitLab.Models.UserToken.CreatedAt.set -> void
 NGitLab.Models.UserToken.ExpiresAt.get -> System.DateTime?
 NGitLab.Models.UserToken.ExpiresAt.set -> void
+NGitLab.Models.UserToken.ExpiresAtStr.get -> string
```

Comment:
I think we can avoid these new props by using a custom converter for DateTime

Reply:
Done